modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItCreateTableDdlTest.java

```diff
@@ -370,4 +371,40 @@ public void testPrimaryKeyIndexTypes() {
                 .returns("TEST3_PK", "HASH", "ID2, ID1")
                 .check();
     }
+
+    @Test
+    public void testSuccessfulCreateTableWithZoneIdentifier() {
+        sql("CREATE ZONE test_zone");
+        sql("CREATE TABLE test_table (id INT PRIMARY KEY, val INT) WITH PRIMARY_ZONE=test_zone");
+        sql("DROP TABLE test_table");
+        sql("DROP ZONE test_zone");
+    }
+
+    @Test
+    public void testSuccessfulCreateTableWithZoneLiteral() {
+        sql("CREATE ZONE test_zone");
+        sql("CREATE TABLE test_table (id INT PRIMARY KEY, val INT) WITH PRIMARY_ZONE='TEST_ZONE'");
+        sql("DROP TABLE test_table");
+        sql("DROP ZONE test_zone");
+    }
+
+    @Test
+    public void testSuccessfulCreateTableWithZoneQuotedLiteral() {
+        sql("CREATE ZONE \"test_zone\"");
+        sql("CREATE TABLE test_table (id INT PRIMARY KEY, val INT) WITH PRIMARY_ZONE='test_zone'");
+        sql("DROP TABLE test_table");
+        sql("DROP ZONE \"test_zone\"");
```

Comment:
let's drop all created zones in @AfterEach section, as we do for tables. Btw, we don't need to drop table in test

Reply:
Done